R/print.effectsize_table.R

```diff
@@ -6,10 +6,34 @@
 #' @inheritParams insight::format_value
 #' @param ... Arguments passed to or from other functions.
 #'
+#' @seealso [insight::display()]
+#'
 #' @export
 print.effectsize_table <- function(x, digits = 2, ...) {
-  x_orig <- x
+  x_fmt <- format(x, digits = digits, ...)
+  cat(insight::export_table(x_fmt, format = NULL, ...))
+  invisible(x)
+}
+
+#' @export
+#' @rdname print.effectsize_table
+print_md.effectsize_table <- function(x, digits = 2, ...) {
+  x_fmt <- format(x, digits = digits, ...)
```

Comment:
you could pass a `format` argument or similar in `format()` methods, so you can generate a proper footer/title for the table (cf. https://github.com/easystats/insight/issues/508)

Reply:
Yeah, but I wanted the lazy way out first :P